lib/cog_api/http/base.ex

```diff
@@ -52,7 +52,8 @@ defmodule CogApi.HTTP.Base do
 
   def find_id_by(%Endpoint{}=endpoint, resource, find_fun)
       when is_function(find_fun) do
-    with {:ok, %{^resource => items}} <- get(endpoint, resource) do
```

Comment:
Is this function being used by anything? I don't think _any_ of these `_by` functions are being used from `HTTP.Base`. Maybe they should be removed?